test/interp/float/float.bril

```diff
@@ -1,4 +1,5 @@
-@main {
+# ARGS: 3 3.0
+@main(float_one : float, float_two : float) {
```

Comment:
Maybe a test for `float`-typed `@main` arguments should be a separate test? (I'm imagining a future student coming along and wanting to implement `float` functionality one step at a time, and maybe a simpler example without `@main` arguments would be nice to have separate from this.)

Reply:
Sure